core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/ExperimentalFeaturesTest.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+package org.eclipse.rdf4j.sail.shacl;
+
+import org.eclipse.rdf4j.model.BNode;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+import org.eclipse.rdf4j.repository.RepositoryException;
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.repository.sail.SailRepositoryConnection;
+import org.junit.AfterClass;
+import org.junit.Test;
+
+/**
+ * @author Håvard Ottestad
+ */
+public class ExperimentalFeaturesTest {
+
+	SimpleValueFactory vf = SimpleValueFactory.getInstance();
+	IRI ex_knows = vf.createIRI("http://example.com/ns#knows");
+	IRI ex_Person = vf.createIRI("http://example.com/ns#Person");
+
+	@AfterClass
+	public static void afterClass() {
+		GlobalValidationExecutionLogging.loggingEnabled = false;
+	}
+
+	@Test
+	public void testDash() throws Exception {
+
+		SailRepository shaclRepository = Utils.getInitializedShaclRepository("test-cases/class/allSubjects/shacl.ttl",
+				false);
+
+		try (SailRepositoryConnection connection = shaclRepository.getConnection()) {
+			connection.begin();
+			connection.add(vf.createBNode(), ex_knows, vf.createBNode());
```

Comment:
This test contains no assertions - what exactly are we testing here? Perhaps clarify with a comment.

Reply:
There are two sets of tests that check that the experimental features are disabled by default and that you can enable them after you've added you shapes. Essentially the tests are "without the experimental features enabled there should be no shacl exception".

Reply:
I will add a comment to the test